src/clj/imcljs/internal/io.clj

```diff
@@ -69,17 +70,31 @@
 
 (defn post-body-wrapper-
   [path {:keys [root token model]} options & [xform]]
-  (parse-response xform (client/post (url root path)
-                                     (-> options ; Blank request map
-                                        ;(wrap-accept)
-                                         (wrap-request-defaults xform) ; Add defaults such as with-credentials false?
-                                        ; If we have basic auth options then convert them from the cljs-http to clj-http format
-                                         (wrap-post-defaults options model)
-                                         wrap-basic-auth
-                                        ;(wrap-post-defaults options model) ; Add form params
-                                         (wrap-auth token)
-                                         (merge {:as :json})))))
-
+  (parse-response (or xform identity)
+                  (client/post (url root path)
+                               (-> options ; Blank request map
+                                  ;(wrap-accept)
```

Comment:
Am I right in thinking these commented lines are legacy stuff that should probably be deleted? If so, let's not copy it over to new methods.